packages/client/package.json

```diff
@@ -35,7 +35,7 @@
     "format": "npm run _format -- --write",
     "format:check": "npm run _format -- --check",
     "test": "vitest",
-    "verify": "npm run typecheck && npm run lint && npm run format:check && npm run test",
+    "verify": "npm run typecheck && npm run lint && npm run format:check && npm run test -- run",
```

Comment:
the verify should always complete .. on CI it does already, ofc, but locally too